M/mlpack/build_tarballs.jl

```diff
@@ -8,12 +8,8 @@ using BinaryBuilder
 name = "mlpack"
 version = v"3.3.0"
 sources = [
-    # Current git master branch as of 12/20/2019.
-    # This will be replaced with the actual mlpack 3.3.0 release... but that
-    # release is dependent on coherent Julia support, so this has to come
-    # first...
-    "https://www.ratml.org/misc/mlpack-3.3.0-a1.tar.gz" =>
-    "70b386c191465feff93d63ac299612e93eb943ea9144525108674ada037321cf"
+    "https://www.mlpack.org/files/mlpack-3.3.0.tar.gz" =>
+    "63cdc3569f2e929899cc30c2e808a42709723c2ea56f8c2953edc7188eab5559"
```

Comment:
While we're changing this line, we've introduced [new types for the sources](https://juliapackaging.github.io/BinaryBuilder.jl/dev/reference/#BinaryBuilder.AbstractSource), to have more control on how they're set up:
```suggestion
    ArchiveSource("https://www.mlpack.org/files/mlpack-3.3.0.tar.gz",
                  "63cdc3569f2e929899cc30c2e808a42709723c2ea56f8c2953edc7188eab5559")
```

Reply:
Sounds good to me. :+1: